### ISSUE TITLE
k0s reset fails to userdel kube-apiserver and other errors

### ISSUE THREAD

**Issue by matti** [k0sproject/k0s#878]
```
$ k0s reset
INFO[2021-04-30 13:37:01] Uninstalling the k0s service
INFO[2021-04-30 13:37:01] no config file given, using defaults
INFO[2021-04-30 13:37:01] deleting user: etcd
INFO[2021-04-30 13:37:01] deleting user: kube-apiserver
userdel: user kube-apiserver is currently used by process 15010
INFO[2021-04-30 13:37:01] deleting user: konnectivity-server
INFO[2021-04-30 13:37:01] deleting user: kube-scheduler
INFO failed to delete controller users: failed to run command "userdel" "kube-apiserver": exit status 8
INFO[2021-04-30 13:37:01] starting containerd for cleanup operations...
INFO error while attempting to clean up worker resources: failed to start containerd: fork/exec /var/lib/k0s/bin/containerd: no such file or directory
INFO[2021-04-30 13:37:01] deleting k0s generated data-dir (/var/lib/k0s) and run-dir (/run/k0s)
INFO[2021-04-30 13:37:01] k0s cleanup operations done. To ensure a full reset, a node reboot is recommended.
```

**Comment by jasmingacic** [comment]
Can you provide more details on this?

**Comment by jasmingacic** [comment]
The behavior of `reset` has been addressed in #928 but it will fail with the same error if one of the "controller" users are "busy"
